taskcluster/ci/ruff/kind.yml

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+kind-dependencies:
+    - docker-image
+
+# Copy of tox job, but doesn't use caching
+
+transforms:
+    - scriptworker_taskgraph.transforms.python_version:transforms
+    - scriptworker_taskgraph.transforms.tox:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+jobs:
+    check:
```

Comment:
I'd prefer to call the kind `check` and the task `ruff` (so it'll be `check-ruff` instead of `ruff-check`). That way if we want to add more of these kinds of tasks in the future, there will be a good place for them.